components/ContributorsList.vue

```diff
@@ -19,6 +19,8 @@
 <script>
 import { leads } from "../data/contributors.json";
 
+var j = 0;
```

Comment:
The linter is catching these: https://github.com/ourjapanlife/findadoc-frontend/pull/217/checks?check_run_id=2978359608